### PR TITLE
FIX-#2225: update parameters of ray APIs for cloud features

### LINKED ISSUE #2225: Update ray API used for cloud features after pin ray>=1.0.0
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- **Modin version** (`modin.__version__`): `0.8.1.1`
- **Python version**:
- **Code we can use to reproduce**:

### Describe the problem

```shell
...
modin.experimental.cloud.base.CannotSpawnCluster: cause: create_or_update_cluste
r() got an unexpected keyword argument 'log_old_style'
Cannot spawn cluster
Cannot destroy cluster:
Traceback (most recent call last):
  File "modin/modin/experimental/cloud/rayscale.py", line 2
20, in __do_destroy
    teardown_cluster(
TypeError: teardown_cluster() got an unexpected keyword argument 'log_old_style'
...
```

### PR DESCRIPTION
**Pull request by anmyachev**
Signed-off-by: Anatoly Myachev <anatoly.myachev@intel.com>

## What do these changes do?

- [ ] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/contributing.html)
- [ ] passes `flake8 modin`
- [ ] passes `black --check modin`
- [ ] signed commit with `git commit -s` 
- [x] Resolves #2225 
- [ ] tests added and passing

### CODE REVIEW

File: modin/experimental/cloud/test/test_cloud.py [hunk 1]
```diff
@@ -14,8 +14,88 @@
...
+    sig_get_head_node_ip = signature(get_head_node_ip)
+
+    def get_head_node_ip_mock(*args, **kwargs):
+        sig_get_head_node_ip.bind(*args, **kwargs)
+        return "127.0.0.1"
+
+    def bootstrap_mock(config):
+        config["auth"]["ssh_user"] = "modin"
+        config["auth"]["ssh_private_key"] = "X" * 20
+        return config
```
Comment (vnlitvinov) [inline modin/experimental/cloud/test/test_cloud.py:51]: these things look like they should be `pytest.fixture` (and other mocks used in other tests)

**Comment by anmyachev** [comment]
@vnlitvinov PR is ready for review.

File: modin/experimental/cloud/test/test_cloud.py [hunk 2]
```diff
@@ -14,8 +14,111 @@
...
+    ):
+        RayCluster(Provider(name="aws"))
+
+
+def test_get_head_node_ip(make_bootstrap_config_mock, make_get_head_node_ip_mock):
+    with mock.patch(
+        "modin.experimental.cloud.rayscale._bootstrap_config",
+        make_bootstrap_config_mock,
+    ):
+        ray_cluster = RayCluster(Provider(name="aws"))
```
Comment (vnlitvinov) [inline modin/experimental/cloud/test/test_cloud.py:86]: this looks like it should be a fixture (using another fixture), as it's repeated quite a while in tests

**Review by vnlitvinov — APPROVED** [review]
Mostly fine, a few notes from me. Nothing blocking, so approval is there, but I would like those fixed.

**Review by vnlitvinov — APPROVED** [review]

**Pull request merged**